.github/workflows/unit_test.yaml

```diff
@@ -1 +1,30 @@
 name: unit test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup conda env
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: test
+          python-version: "3.8"
+      - name: Install dependencies
+        shell: bash -l {0}
+        run: |
+          conda activate test
+          conda install pytorch torchvision torchtext cpuonly -c pytorch-nightly
+          pip install -r requirements.txt
+      - name: Run unit tests
+        shell: bash -l {0}
+        run: |
+          conda activate test
+          pytest test
```

Comment:
Just wanna confirm, will this signal still pass with our skipped unit tests?

Reply:
its already skipping (see the tp link)

Reply:
Can we increase the versobity `pytest -vv`?

Reply:
Can we increase the verbosity `pytest -vv`?

Reply:
Can we increase the verbosity `pytest -vv`?